Blaise-Shyaka/My-Mentor
# [finishes #170422756] complete mentor-feedback page

# What does this PR do?

The page allows the mentor to send feedback to the mentee

# Description of tasks to be completed

- The page should contain a form where the mentor types his/her feedback and a button to submit
- The page should be responsive

# How should this be manually tested?

- Clone this repository and then cd into views
- Open mentor-feedback.html 
- Check its responsiveness and a simple form allowing the mentor to submit her feedback
